tests/Inspetor/Model/InspetorSaleTest.php

```diff
@@ -29,7 +29,7 @@ private function getDefaultItem() {
         $item = new InspetorItem();
         $item->setId("123");
         $item->setEventId("123");
-        $item->setSessionId("123");
+        $item->setSessionIds("123");
```

Comment:
Don't we want to pass it as an array here and in line [161 of following file](https://github.com/inspetor/inspetor-php/pull/31/files#diff-1cd5ab81ebd5c26ca9dbff061936cf6fR161)?

Reply:
Good catch! I added some more stringent checks :)